src/smt/set_defaults.cpp

```diff
@@ -818,8 +818,6 @@ void SetDefaults::setDefaultsPost(const LogicInfo& logic, Options& opts) const
       SET_AND_NOTIFY(Arith, nlCov, true, "QF_UFNRA");
       SET_AND_NOTIFY_IF_NOT_USER(
           Arith, nlExt, options::NlExtMode::LIGHT, "QF_UFNRA");
-      SET_AND_NOTIFY_IF_NOT_USER(
```

Comment:
From your message [here](https://github.com/cvc5/cvc5/issues/9999#issuecomment-1834543022) it sounds like it's not enough to not enable it by default, but maybe we even need to disallow it when nl-cov is enabled?

Reply:
It's an expert option, so this isn't necessary at the moment.